cassandra/protocol.py

```diff
@@ -682,9 +686,14 @@ def recv_results_metadata(cls, f, user_type_map):
         else:
             paging_state = None
 
+        if flags & 0x0008:
```

Comment:
better to create a constant var rather than using the hexa number directly here

Reply:
Yep, thanks for catching this

Reply:
Addressed :+1: